Tasks/google-play-release/task.json

```diff
@@ -55,7 +55,7 @@
             "label": "APK Path",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "Path to the APK file you want to publish to the specified track."
+            "helpMarkDown": "Path to the APK file you want to publish to the specified track. Wildcards can be used. For example, **/*.apk to match the first APK file."
```

Comment:
Just took a look at how the Gradle task formats their glob example, and apparently VSTS supports using back ticks to mark code samples. Can we wrap the **/*.apk span in back ticks to get the same formatting in the UI? Also, can we add ", in any directory." to the end of this description, just to call out what the ** part means.